easybuild/easyconfigs/g/GetOrganelle/GetOrganelle-1.7.2-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'GetOrganelle'
+version = '1.7.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Kinggerm/GetOrganelle'
+description = """This toolkit assemblies organelle genome from genomic skimming data."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/Kinggerm/GetOrganelle/archive/%(version)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['54923ca971d68620aaf036a324b554084b00f7c209b0b3a234f815c82ee6d079']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('Bandage', '0.8.1', '_Centos', SYSTEM),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('sympy', '1.6.2', versionsuffix),
+    ('SPAdes', '3.14.1', versionsuffix),
+    ('Bowtie2', '2.4.1'),
+    ('BLAST+', '2.10.1'),
+    ('Perl', '5.30.2'),
+    ('matplotlib', '3.2.1', versionsuffix)
+]
+
+options = {'modulename': False}
+
+sanity_check_commands = ["get_organelle_from_reads.py -h"]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```

Comment:
This is the default for `PythonPackage`, so unless there's something else to check for, just remove `sanity_check_paths`

Reply:
ok